cilium/cmd/fixtures/proc/net/xfrm_stat

```diff
@@ -0,0 +1,29 @@
+XfrmInError                     1
+XfrmInBufferError               2
+XfrmInHdrError                  4
+XfrmInNoStates                  3
+XfrmInStateProtoError           40
+XfrmInStateModeError            100
+XfrmInStateSeqError             6000
+XfrmInStateExpired              4
+XfrmInStateMismatch             23451
+XfrmInStateInvalid              55555
+XfrmInTmplMismatch              51
+XfrmInNoPols                    65432
+XfrmInPolBlock                  100
+XfrmInPolError                  10000
+XfrmOutError                    1000000
+XfrmOutBundleGenError           43321
+XfrmOutBundleCheckError         555
+XfrmOutNoStates                 869
+XfrmOutStateProtoError          4542
+XfrmOutStateModeError           4
+XfrmOutStateSeqError            543
+XfrmOutStateExpired             565
+XfrmOutPolBlock                 43456
+XfrmOutPolDead                  7656
+XfrmOutPolError                 1454
+XfrmFwdHdrError                 6654
+XfrmOutStateInvalid             28765
+XfrmAcquireError                24532
+XfrmOutStateInvalid             28765
```

Comment:
Was this file copied from https://github.com/prometheus/procfs/blob/5162bec877a860b5ff140b5d13db31ebb0643dd3/xfrm_test.go? If that's the case, I think we'll need to retain the copyright notice and add our own copyright.

Reply:
Hi Paul! Actually this file was generated from fixtures.ttar in the prometheus library using `make tests`. And so just used the values in the function as here https://github.com/prometheus/procfs/blob/5162bec877a860b5ff140b5d13db31ebb0643dd3/xfrm_test.go. Adding copyright notice to the top of the file would cause parsing errors. Any hints on this ?

Extending my doubt I borrowed the function here https://github.com/h3llix/cilium/commit/c7fb4e64ff382eb0dd461932c861a8a0eae23e38#diff-40188efe162d2c3f2a87bfb4ac62a4fe8e202acc34e15dc44fe5322fb74dad68R42 from netlink library but just modified it a bit(not committed but will update it  in the final patch). So, in this case do I need to change the copyright notice ?

Reply:
> Actually this file was generated from fixtures.ttar in the prometheus library using make tests. And so just used the values in the function as here https://github.com/prometheus/procfs/blob/5162bec877a860b5ff140b5d13db31ebb0643dd3/xfrm_test.go. Adding copyright notice to the top of the file would cause parsing errors. Any hints on this ?

I've generated the following file on my system. You can use it instead of the other.
```
XfrmInError             	2
XfrmInBufferError       	0
XfrmInHdrError          	0
XfrmInNoStates          	225479
XfrmInStateProtoError   	141222
XfrmInStateModeError    	0
XfrmInStateSeqError     	0
XfrmInStateExpired      	0
XfrmInStateMismatch     	0
XfrmInStateInvalid      	0
XfrmInTmplMismatch      	0
XfrmInNoPols            	203389
XfrmInPolBlock          	0
XfrmInPolError          	0
XfrmOutError            	0
XfrmOutBundleGenError   	0
XfrmOutBundleCheckError 	0
XfrmOutNoStates         	36162
XfrmOutStateProtoError  	1886
XfrmOutStateModeError   	0
XfrmOutStateSeqError    	0
XfrmOutStateExpired     	0
XfrmOutPolBlock         	0
XfrmOutPolDead          	0
XfrmOutPolError         	0
XfrmFwdHdrError         	0
XfrmOutStateInvalid     	0
XfrmAcquireError        	0
```

> Extending my doubt I borrowed the function here h3llix@c7fb4e6#diff-40188efe162d2c3f2a87bfb4ac62a4fe8e202acc34e15dc44fe5322fb74dad68R42 from netlink library but just modified it a bit(not committed but will update it in the final patch). So, in this case do I need to change the copyright notice ?

Yes, you need to add the copyright notice if it's the same license. If it's not the same license, then it may get more complicated.

Reply:
I just changed the function but Can you just verify that its ok now ?

Reply:
The file is missing a newline at the end, but other than that it looks okay (including the function).

Reply:
Thanks! Added a newline as well rebased on latest master.